templates/docs/examples/patterns/search-box/default-dark.html

```diff
@@ -5,14 +5,16 @@
 
 {% block content %}
 <form class="p-search-box is-dark">
-  <input type="search" class="p-search-box__input" placeholder="Search" name="search" autocomplete="on" required>
-  <button type="reset" class="p-search-box__reset"><i class="p-icon--close"></i></button>
-  <button type="submit" class="p-search-box__button"><i class="p-icon--search"></i></button>
+  <label class="u-off-screen" for="search">Search</label>
```

Comment:
Wouldn't it be enough to add `aria-label` to the search input? To avoid adding hidden element?

Reply:
I took a look at what we already do on search and filter, which was hiding the label element. I wanted to keep consistency, but not being sure I did a little bit of digging and came across this article. It argues for hiding `labels` rather than adding `aria-label`s due to some browser + assistive tech combination inconsistencies.

https://gomakethings.com/revisting-aria-label-versus-a-visually-hidden-class/#:~:text=In%20the%20posts%2C%20I%20recommended,visually%20shown%20to%20sighted%20users.

What do you think?

Reply:
That's fine. If there is some evidence that keeping label separate may be better let's keep it.